.github/workflows/version.yml

```
@@ -126,3 +118,26 @@ jobs:
           title: '[AUTO] [main] Add bwc version ${{ env.NEXT_VERSION }}.'
           body: |
             I've noticed that a new tag ${{ env.TAG }} was pushed, and added a bwc version ${{ env.NEXT_VERSION }}.
+
+      - name: Create tracking issue
+        id: create-issue
+        uses: actions/github-script@v6.4.0
+        with:
+          script: |
+            const body = `
+            Issue for tracking the changes associated with a version increment.
+            Tracking PRs:
+            - [ ] ${{ steps.base_pr.outputs.pull-request-url }}
+            - [ ] ${{ steps.base_x_pr.outputs.pull-request-url }}
+            - [ ] ${{ steps.main_pr.outputs.pull-request-url }}
```

Lets follow our issue format and provide context for folks if this is their first time seeing this issue, what do you think of the following?

```suggestion
            ### Description
            A new version of OpenSearch was released, to prepare for the next release new version numbers need to be updated in all active branches of development.

            ### Exit Criteria
            Review and merged the following pull requests
            - [ ] ${{ steps.base_pr.outputs.pull-request-url }}
            - [ ] ${{ steps.base_x_pr.outputs.pull-request-url }}
            - [ ] ${{ steps.main_pr.outputs.pull-request-url }}
            
            ### Additional Context
            See project wide guidance on branching and versions [[link]](https://github.com/opensearch-project/.github/blob/main/RELEASING.md).
```

---

Updated.